src/lib/cache.ts

```diff
@@ -68,19 +60,17 @@ function createMockClient() {
 }
 
 function createMemcachedClient() {
-  const client = new Memcached(MEMCACHED_URL, {
-    poolSize: MEMCACHED_MAX_POOL,
-  })
-  VerboseEvents.forEach((event) => {
-    client.on(event, () => verbose(`[Cache] ${event}`))
+  console.log("hey")
```

Comment:
was just making sure this was a singleton

Reply:
👋